cloudesire-api-client-dto/src/main/java/com/cloudesire/platform/apiclient/dto/model/dto/BudgetDTO.java

```diff
@@ -288,11 +303,29 @@ public void setVmConfiguration( BudgetVMConfigurationDTO vmConfiguration )
         this.vmConfiguration = vmConfiguration;
     }
 
+    public Boolean getDeferred()
+    {
+        return deferred;
+    }
+
+    public void setDeferred( Boolean deferred )
+    {
+        this.deferred = deferred;
+    }
+
+    /**
+     * @deprecated by {@link #getDeferred()}
+     */
+    @Deprecated
     public Boolean getDowngrade()
     {
         return downgrade;
     }
 
+    /**
+     * @deprecated by {@link #setDeferred(Boolean)}
+     */
+    @Deprecated
     public void setDowngrade( Boolean downgrade )
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3AS1133)